src/components/pages/Chat/AddRoomModal/PrivacySelect.tsx

```diff
@@ -0,0 +1,185 @@
+import React from 'react'
+import Select, { components } from 'react-select'
+import styled from 'styled-components'
+import publicIcon from './public.svg'
+import viewOnlyIcon from './viewonly.svg'
+import privateIcon from './private.svg'
+import checkIcon from './check.svg'
+
+const options = [
+    {
+        label: 'Private',
+        subLabel: 'Only invited members can post and view messages',
+        value: 'private',
+        icon: privateIcon,
+    },
+    {
+        label: 'View only',
+        subLabel: 'Anyone can view other messages',
+        value: 'viewonly',
+        icon: viewOnlyIcon,
+    },
+    {
+        label: 'Public',
+        subLabel: 'Anyone can post and view other messages',
+        value: 'public',
+        icon: publicIcon,
+    },
+]
+
+const customStyles = {
+    control: (provided: any) => ({
+        ...provided,
+        border: 'none',
+        display: 'flex',
+        flexDirection: 'row',
+        backgroundColor: '#DEE6EE',
+        height: '56px',
+        borderRadius: '8px',
+        alignItems: 'center',
+        paddingLeft: '20px',
+        paddingRight: '10px',
+        marginBottom: '15px',
+    }),
+    indicatorSeparator: () => ({ display: 'none' }),
+    menu: (provided: any) => ({
+        ...provided,
+        borderRadius: '10px',
+        boxShadow: '0px 0px 10px rgba(0, 0, 0, 0.1)',
+        marginTop: '8px',
+        zIndex: '100000',
+    }),
+    option: (provided: any) => ({
+        ...provided,
+        backgroundColor: 'white',
+    }),
+}
+
+const OptionContainer = styled.div`
+    display: flex;
+    flex-direction: row;
+    align-items: center;
+`
+
+const LabelContainer = styled.div`
+    display: flex;
+    flex-direction: column;
+    margin-right: auto;
+`
+
+const IconContainer = styled.div`
+    border-radius: 50%;
+    background-color: #f1f4f7;
+    width: 30px;
+    height: 30px;
+    display: flex;
+    align-items: center;
+    justify-content: center;
+    margin-right: 10px;
+`
+
+const CheckIcon = styled.img`
+    width: 12px;
+    height: 9px;
+    margin-left: auto;
+`
+
+const ControlIcon = styled.div`
+    color: #59799C:
+`
+
+const Option = (props: any) => {
+    return (
+        <components.Option {...props}>
+            <OptionContainer>
+                <IconContainer>
+                    <img src={props.data.icon} />
+                </IconContainer>
+                <div
+                    style={{
+                        display: 'flex',
+                        flexDirection: 'row',
+                        alignItems: 'center',
+                        justifyContent: 'space-between',
+                        width: '100%',
+                    }}
```

Comment:
I can live with that but would prefer to either
- make them stand-alone styled components (avoid `style`) – that's the preferred way, or
- use [`css` property](https://styled-components.com/docs/api#css-prop).